### PR TITLE
docs: readme formatting

### PR DESCRIPTION
**Pull request by grant**
## Description

Fixes https://github.com/GoogleCloudPlatform/python-docs-samples/issues/4837

- Formats the README
  - (Similar to the markdown in the Node repo)
- Includes generic how to run instructions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,68 @@
...
+
+1. Run the sample:
+
+    ```
+    python snippets.py
+    ```
+
+## Contributing
+
+Contributions welcome! See the [Contributing Guide][CONTRIBUTING.md].
```
Comment (engelke) [inline README.md:58]: Need parens, not square brackets, around CONTRIBUTING.md.

**Review by engelke — APPROVED** [review]
LGTM. Thanks for this!

File: README.md (hunk 1 above)
Comment (grant) [reply README.md]: Fixed. :)

**Review by leahecole — COMMENTED** [review]
LGTM - just want to double check with the other @GoogleCloudPlatform/python-samples-owners - this isn't a generated file, right? (I'm mostly certain it's not generated in this repo) and as long as it's not, I'm cool with it. Thank you for doing this, @grant !

File: README.md [hunk 2]
```diff
@@ -1,16 +1,67 @@
...
+
+## How to run a sample
+
+1. Change directory to one of the sample folders, e.g. `logging/cloud-client`:
+
+    ```
+    cd logging/cloud-client/
+    ```
+
+1. Create a virtualenv. Samples are compatible with Python 2.7 and 3.4+.
```
Comment (busunkim96) [inline README.md:36]:
```suggestion
1. Create a virtualenv. Samples are compatible with Python 3.6+.
```

File: README.md [hunk 3]
```diff
@@ -1,16 +1,67 @@
...
+
+    ```
+    virtualenv env
+    source env/bin/activate
+    ```
```
Comment (busunkim96) [inline README.md:41]:
```suggestion
    ```
    python3 -m venv env
    source env/bin/activate
    ```
```

**Comment by busunkim96** [comment]
> LGTM - just want to double check with the other @GoogleCloudPlatform/python-samples-owners - this isn't a generated file, right? (I'm mostly certain it's not generated in this repo) and as long as it's not, I'm cool with it. Thank you for doing this, @grant !

I can confirm this one isn't generated.

File: README.md [hunk 4]
```diff
@@ -1,16 +1,67 @@
...
+
+## How to run a sample
+
+1. Change directory to one of the sample folders, e.g. `logging/cloud-client`:
+
+    ```
+    cd logging/cloud-client/
+    ```
+
+1. Create a virtualenv. Samples are compatible with Python 3.6+.
```
Comment (engelke) [inline README.md:36]: Samples are compatible with Python 3.6+, except for those in the appengine/standard folder, which are compatible with Python 2.7.

Comment (grant) [reply README.md:36]: Do you want me to state that in the main README with dozens of products? That seems like an exception, in which case we can document it in that sub sub dir.

https://github.com/GoogleCloudPlatform/python-docs-samples/tree/master/appengine/standard

I think it's fine, but we could add "Samples are **generally** compatible with Python 3.6+." - but that's leaving a bit of unnecessary mystery imo.

Comment (engelke) [reply README.md:36]: I'm okay with this.

**Comment by grant** [comment]
Please merge when approved, or request changes. :)

**Review by leahecole — APPROVED** [review]

**Pull request merged**